playwright/tests/titleSearch.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { Given, Then, When } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given("I want to do a title search", async function (this: CustomWorld) {
+  return await this.page.goto("https://digital-research-books-beta.nypl.org/");
```

Comment:
should we be doing these tests against QA?

Reply:
Chris is going to create a separate ticket for this change 👍